src/odemis/gui/model/__init__.py

```diff
@@ -1042,6 +1045,12 @@ def __init__(self, main):
         # For dichotomic mode
         self.dicho_seq = model.ListVA()  # list of 4 enumerated for each corner
 
+class EnzelAlignGUIData(ActuatorGUIData):
+    def __init__(self, main):
+        ActuatorGUIData.__init__(self, main)
+        self.viewLayout.value = VIEW_LAYOUT_VERTICAL
+        self.step_size = model.FloatVA(1e-6)
```

Comment:
Please make it a FloatContinuous, with a range and a unit. If you don't know the range, pick something "safe" like (1e-9 -> 1e-3).